packages/currency/src/chains/evm/mainnet.ts

```diff
@@ -0,0 +1,7 @@
+import { TokenMap } from '../../types';
```

Comment:
This file structure should be referenced in the `README` as it looks key.

Reply:
What do you recommend we describe? The file structure is [enforced by TS](https://github.com/RequestNetwork/requestNetwork/blob/05ab7d5ef587de53bd2d8c3e3fb8181586535ef6/packages/currency/src/chains/evm/index.ts#L26).

Reply:
I actually added some info in [3452fd1](https://github.com/RequestNetwork/requestNetwork/pull/1056/commits/3452fd1a3f06b413a44d9406b60b271dacd2f206) because the `testnet: true` property that is introduced in https://github.com/RequestNetwork/requestNetwork/pull/1048 is not obvious and not a required property. Let me know what you think.